content/influxdb/cloud/account-management/pricing-plans.md

```diff
@@ -25,18 +25,28 @@ InfluxDB Cloud offers two pricing plans:
 
 ## Free Plan
 
-All new {{< cloud-name >}} accounts start with a rate-limited Free Plan.
-Use this plan as much and as long as you want within the Free Plan rate limits:
+All new {{< cloud-name >}} accounts start with Free Plan that limits data and resource usage.
+Use this plan as much and as long as you want within the Free Plan limits:
 
-#### Free Plan rate limits
+### Free Plan limits
+
+The Free Plan includes the rate limits, data limits, and resource limits below.
+
+### Rate limits
+
+- **Data In:** .017MB per second
+- **Query:** 1MB per second
+
+### Data limits
```

Comment:
Distinguished between "rate limits," "data limits," and "resource usage"...not married to any of the terminology, just a start to distinguish differences. Also, we were showing data in/query rate limits in spans of 5 minutes, so converted to per second rate. Discussed with @corriganjohn who vaguely recalled this original verbiage may have come from @rkramer9967? Prob best to align terms and verbiage cross our business communications, docs, and UI to the extent possible. Will stay tuned.

Reply:
The original verbiage pre-dates my tenure here at Influx. My understanding is the system truly monitors the rate limits in cumulative  5 minute increments as described on our website and so our documentation should continue to align to Data In 5MB/5 minutes and Query 300MB/5 minutes rather than converting these to their per second equivalents, i.e. the system does not limit to .017MB per second of ingest. I can ingest at a higher rate than .017mb/sec for a period of time subject to not exceeding 300MB in a 5 minute period.

Reply:
Excellent feedback, thank you, @rkramer9967. Will leave as is if that's how the system currently works. Hoping for dev to confirm.

Background: Issue with docs surfaced yesterday as @corriganjohn was testing limits. Chatted with @corriganjohn who noticed potential UX confusion conveying input/out metrics in per/second some places and per 5 minutes here. Understand why we don't want to enforce limit per second.

Thoughts on expanding text to explicitly say: _Although InfluxDB supports tracking data ingest/query per/sec (or more precise level?), InfluxDB only checks ingest/query limits in five minute interrvals, and then enforces the limit if specifed ingest/query data exceeds MB in the previous 5 minutes._

Reply:
I don't think this is entirely accurate, based on testing. But not sure how detailed the explanation needs to be for the users. As far as I can tell (this needs development to confirm): 

- The system takes the assigned rate limit value (measured in per second) and multiplies that (by the number of seconds in 5 minutes) to arrive at a 'limit value'.
- Every 5 minutes the 'tracking' is cleared and restarted
- If the rate limit is hit at any point within the time window (eg it might only take 1 minute of intense processing) then all processes are stopped. No NEW processes can be triggered until the 'clock' is reset (after 5 minutes).
- Once the clock is reset, new in/out processes can be triggered (processes that were running when limit was hit do NOT restart. Note: this could potentially affect any Tasks that were set to run during the 'time out' window)

While we set the limit metric using 'per second', i think it makes sense to be consistent and only use the 'per 5 minute' calculated value for the users